init.go

```diff
@@ -88,6 +89,22 @@ func main() {
 	if err := runScripts(run, allScripts, scriptEnvVar); err != nil {
 		log.Fatalf("init failed: %v", err)
 	}
+
+	node, err := kubeclient.CoreV1().Nodes().Get(os.Getenv("NODE"), v1.GetOptions{})
+	if err != nil {
+		log.Fatalf("failed to get node from name \"%s\": %v", os.Getenv("NODE"), err)
+	}
+
+	file, err := os.Create("/var/lib/storageos/init_envs.sh")
+	if err != nil {
+		log.Fatalf("failed to create file init_envs.sh: %v", err)
+		os.Exit(1)
```

Comment:
nit: `log.Fatal` will already call `os.Exit`, so there's no need for the extra one here

Reply:
Done